polaris-react/src/styles/foundation/_focus-ring.scss

```diff
@@ -1,7 +1,7 @@
 /// Sets the focus ring for an interactive element
 /// @param {String} $size - The size of the border radius on the focus ring.
 /// @param {String} $style - Focus ring state.
-/// @param {Number} $border-width - The border width of your element in rems.
```

Comment:
Not just used to offset the `border-width` of an element—it's also just sometimes used as a negative offset and so the description here felt a bit misleading (the focus ring around the mobile nav icon is a great example of this).